core/src/main/scala/cats/data/Func.scala

```diff
@@ -113,7 +113,7 @@ sealed private[data] trait FuncApply[F[_], C] extends Apply[λ[α => Func[F, C,
 sealed private[data] trait FuncApplicative[F[_], C] extends Applicative[λ[α => Func[F, C, α]]] with FuncApply[F, C] {
   def F: Applicative[F]
   def pure[A](a: A): Func[F, C, A] =
-    Func.func(c => F.pure(a))
+    Func.func(_ => F.pure(a))
```

Comment:
should we use `Function.const` for these cases?

Reply:
I personally would love to. I just thought it is not appreciated in Cats for some reason. But if you think it is fine, I'll change to `const` for sure.

Reply:
used